include/ibm/management_console_rest.hpp

```diff
@@ -55,9 +55,42 @@ static boost::container::flat_map<std::string,
                                   std::unique_ptr<sdbusplus::bus::match::match>>
     ackMatches;
 
-inline void saveConfigFile(const std::string& data, const std::string& fileID,
-                           const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
-                           std::uintmax_t saveAreaDirSize)
+inline bool isValidConfigFileName(const std::string& fileName,
```

Comment:
Why are we moving this function ?

Reply:
Probably it is just to declare the method before its usage, to avoid "method not declared in the scope" error